tests/migration_test.go

```diff
@@ -1016,7 +1016,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			It("should migrate vmi with a usb disk", func() {
 
 				vmi := libvmi.NewAlpineWithTestTooling(
-					libvmi.WithEmptyDisk("disk0", v1.DiskBusUSB, resource.MustParse("128Mi")),
+					libvmi.WithEmptyDisk("uniqueusbdisk", v1.DiskBusUSB, resource.MustParse("128Mi")),
```

Comment:
how about panicking if a disk name is in use instead of doing nothing? It never sounds like the right decision to do nothing. Override would make more sense than doing nothing in general, but it would be impossible to differentiate between a desired override or an unintentional one.

In theory adding an `Options` struct to `WithEmpytDisk` which may allow setting a DesiredOverrideWhichHasToHappen option may allowing introducing an override  in a safe way, but just panicking seems to be what we want to do right now?

Reply:
@rmohr Hey Roman, I agree with your points. There is https://github.com/kubevirt/kubevirt/pull/8348 that is trying to address it but there are multiple opinions. Maybe you want to put this opinion there.